integrationTest/integration_test.go

```diff
@@ -458,6 +458,7 @@ func makeTests(t *testing.T) []*TestCase {
 			tc("Change Priority", srv("_sip._tcp", 52, 6, 7, "foo.com."), srv("_sip._tcp", 15, 65, 75, "foo4.com.")),
 			tc("Change Weight", srv("_sip._tcp", 52, 62, 7, "foo.com."), srv("_sip._tcp", 15, 65, 75, "foo4.com.")),
 			tc("Change Port", srv("_sip._tcp", 52, 62, 72, "foo.com."), srv("_sip._tcp", 15, 65, 75, "foo4.com.")),
+			tc("Null Target", srv("_sip._tcp", 52, 62, 72, "foo.com."), srv("_sip._tcp", 15, 65, 75, ".")),
```

Comment:
This may break other providers if they don't handle empty SRV records properly.

Reply:
Did you see this in the wild?  It seems like a bad thing to do, even if it is legal.  Like, maybe we should add a new opinion that bans this kind of thing (not because it is illegal, but because it is ill-advised).

Reply:
Yes, I've seen it in the wild (both its usage and providers not supporting it, I don't know which you're referring to). For example, [Fastmail](https://www.fastmail.com/help/receive/domains-advanced.html) and [Gmail](https://dns.google.com/resolve?name=_pop3._tcp.gmail.com&type=SRV) both use this for pop3. I've also seen it for other services like unencrypted SIP ports on VoIP providers, but I don't recall exactly which ones at the moment. For providers not supporting it, I have heard Plesk didn't support it.

Reply:
I shouldn't read Gihub from my phone. I totally missed that "." is a legal target (you had written about that above). My apologies.

Reply:
No worries.